helpdesk_mgmtsystem_nonconformity/models/mgmtsystem_nonconformity_stage.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MgmtsystemNonconformityStage(models.Model):
+    _inherit = "mgmtsystem.nonconformity.stage"
+
+    helpdesk_stage_ids = fields.One2many(
```

Comment:
Why a one2many? Maybe you have to restrict the possibility of linking to more than one stages. If not, the stage auto-switch may be not correct.